src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -40,7 +40,23 @@
         There are no questions for you to answer here!
       </c:when>
       <c:otherwise>
-        <input type="checkbox" name="sendsubmissionemail">
+        <c:set var="isResponsePresent" value="false" scope="page"/>
```

Comment:
Can we encapsulate this function to the data object, namely `FeedbackSubmissionEditPageData`? There should be no much logic inside the a `jsp` file.

Reply:
Thanks for pointing this out :) It did bugged me writing logic in the `jsp` file but I focused on making it work first and ultimately neglected this aspect :P

There are two questions I would like to ask with regards to working with `jsp` files. 
1. Is there a recommended workflow for debugging `jsp` files in TEAMMATES?
2. In the new `feedbackSubmissionForm.tag` file that I committed, I realized that `data.responsePresent` corresponds to the method `isResponsePresent` written in `FeedbackSubmissionEditPageData`. Is this the work of `jsp` recognizing `is` and any other similar keywords such that we can omit the full method name `isResponsePresent`? E.g. `data.isResponsePresent` does not work but `data.responsePresent` works.

Reply:
The EL (Expression Language) in JSP (such as `{data.xxxx}`) replies on the JavaBeans specification when it comes to accessing properties. The specification is here http://download.oracle.com/otndocs/jcp/7224-javabeans-1.01-fr-spec-oth-JSpec/ if u are instered. For example, it says that you can access the boolean property using `{data.good}` if u have defined a method (getter) called `isGood()` in the class